api/controllers/HeartbeatController.js

```diff
@@ -13,7 +13,7 @@ const formatMaxMinMean = (collection, key) => {
 module.exports = {
   _config: {
     populate: true,
-    limit: 0
+    limit: 9223372036854775807
   },
```

Comment:
The 'limit' you have above configuring which config?  as the sails.config.blueprints uses 'defaultLimit' ... to set the default number of records to show in the response from a "find" action.